lib/rohd_hcl.dart

```diff
@@ -1,10 +1,15 @@
 // Copyright (C) 2023-2024 Intel Corporation
 // SPDX-License-Identifier: BSD-3-Clause
 
-export 'src/adder.dart';
 export 'src/arbiters/arbiters.dart';
+export 'src/arithmetic/adder.dart';
```

Comment:
We can add an "arithmetic.dart" that exports everything that we want to from the arithmetic directory instead of adding all these at the top-level here.

When you do so, review which things we want to export visibly to anyone and which should remain internal to rohd-hcl's implementation.  Anything not exported won't be automatically included when people import rohd_hcl

Reply:
I organized things a bit better to allow us to only export the main classes.  I created an arithmetic.dart file to export.